models/events/webhook/github/GithubPush.js

```diff
@@ -0,0 +1,5 @@
+module.exports = require('../../../Model').extend({
+  properties: {
+    compare: String
```

Comment:
If this is a placeholder, it might be a good idea change this to become a `TestEvent` used specifically for testing since it's already been added to the tests below.